tests/page-objects/messages/sms-gateway.po.js

```diff
@@ -0,0 +1,114 @@
+const utils = require('../../utils');
+const commonElements = require('../common/common.po');
+const helper = require('../../helper');
+const { browser, element } = require('protractor');
+
+//page objects        
+const  sentTask= element(by.css('#reports-content .details > ul .task-list .task-state .state'));
+   
+const deliveredTask = element(by.css(
+  '#reports-content .scheduled-tasks > ul > li:nth-child(1) > ul > li:nth-child(1) .task-state .state'));
+
+const scheduledTask = element(by.css(
+  '#reports-content .scheduled-tasks > ul > li:nth-child(1) > ul > li:nth-child(2) .task-state .state'));
+      
+const failedTask = element(by.css(
+  '#reports-content .scheduled-tasks > ul > li:nth-child(2) > ul > li:nth-child(1) .task-state .state'));
+    
+const feedback = element(
+  by.css('#reports-content .details > ul .task-list .task-state .state')
+);
+
+const incomingData = element(by.css('#message-content li.incoming:first-child .data p:first-child'));
+
+
+ 
+// scheduled tasks
+// State for messageId2 is still forwarded-to-gateway
+const message2State = element(by.css(
+  '#reports-content .scheduled-tasks > ul > li:nth-child(1) > ul > li:nth-child(1) .task-state .state'));
+
+const getTaskState = element => {
+  helper.waitUntilReady(element);
+  return helper.getTextFromElement(element);
+}; 
+
+module.exports = {
+  
+  showMessageList : async () => {
+    utils.resetBrowser();
+    helper.clickElement(element(by.id('messages-tab')));
+  
+    // LHS
+    helper.waitElementToPresent(
+      element(by.css('#message-list li:first-child'))
+    );
+    browser.waitForAngular();
+    helper.waitElementToBeVisible(
+      element(by.css('#message-list li:first-child'))
+    );
+  }, 
+
+  expectMessage: async (heading, summary) => {
+    await expect(
+      helper.getTextFromElement(
+        element(by.css('#message-list li:first-child .heading h4'))
+      )
+    ).toBe(heading);
+    await expect(
+      helper.getTextFromElement(
+        element(by.css('#message-list li:first-child .summary p'))
+      )
+    ).toBe(summary);
+  },
+
+  showMessageDetails: async () => {
+    // RHS
+    helper.clickElement(
+      element(by.css('#message-list li:first-child .summary'))
+    );
+    helper.waitElementToBeVisible(incomingData);  
+  },
+
+  expectMessageDetails: async (header, text, status) => {
+    browser.waitForAngular();
+    const messageHeader = helper.getTextFromElement(
```

Comment:
Move the awaits to the helper calls and you then can remove them from the expects. 

` const messageHeader = await helper.getTextFromElement(`

Reply:
`const messageText = await helper.getTextFromElement(incomingData);`
 `   const messageStatus = await helper.getTextFromElement(`